docs/src/SUMMARY.md

```diff
@@ -4,7 +4,7 @@
 - [Installation](./getting_started.md)
   - [Ubuntu/Debian](./install/debian.md)
   - [MacOS](./install/macos.md)
-  - [Docker](./install/docker.md)
+  - [Docker (Recommended)](./install/docker.md)
   - [Source](./install/source.md)
```

Comment:
Not sure if docker itself is recommended.. by you sure :) I'm impartial ;)

Reply:
tried to sneak that one by you haha - I'll remove it